starwars_api/models.py

```diff
@@ -55,17 +64,47 @@ def __repr__(self):
 class BaseQuerySet(object):
 
     def __init__(self):
-        pass
+        self.page = 1
+        self.index = 0
+        self.json_data = api_client.get_people(**{'page': 1})
```

Comment:
Why'd you pack the keyword arg into a literal dict just to unpack it again rather than doing ...`get_people(page = 1)`?

(Leftover from a previous strategy you were perusing?)